src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -208,6 +211,14 @@ internal ImmutableArray<NamedTypeSymbol> AllInterfacesNoUseSiteDiagnostics
             }
         }
 
+        internal ImmutableArray<INamedTypeSymbol> AllInterfacesPublicSymbolsNoUseSiteDiagnostics
```

Comment:
>NoUseSiteDiagnostics [](http://example.com/codeflow?start=76&length=20)

The "NoUseSiteDiagnostics" suffix is probably not necessary for this API

Reply:
No longer in the PR